examples/README.md

```diff
@@ -296,3 +296,76 @@ Units: sectors of 1 * 512 = 512 bytes
 Sector size (logical/physical): 512 bytes / 512 bytes
 I/O size (minimum/optimal): 4194304 bytes / 4194304 bytes
 ```
+### How to create Cephfs Snapshots
+
+Before continuing, make sure you enabled the required
+feature gate `VolumeSnapshotDataSource=true` in your Kubernetes cluster.
```

Comment:
This is no more required on the supported kubernetes version and we can skip it?